packages/components/tests/integration/components/hds/tabs/index-test.js

```diff
@@ -7,12 +7,93 @@ module('Integration | Component | hds/tabs/index', function (hooks) {
   setupRenderingTest(hooks);
 
   test('it renders the component', async function (assert) {
-    await render(hbs`<Hds::Tabs />`);
+    await render(hbs`
+      <Hds::Tabs as |T|>
+        <T.Tab>One</T.Tab>
+        <T.Tab>Two</T.Tab>
+        <T.Panel>Content 1</T.Panel>
+        <T.Panel>Content 2</T.Panel>
+      </Hds::Tabs>
+    `);
     assert.dom(this.element).exists();
   });
 
   test('it should render with a CSS class that matches the component name', async function (assert) {
-    await render(hbs`<Hds::Tabs id="test-tabs" />`);
+    await render(hbs`
+      <Hds::Tabs id="test-tabs" as |T|>
+        <T.Tab>One</T.Tab>
+        <T.Tab>Two</T.Tab>
+        <T.Panel>Content 1</T.Panel>
+        <T.Panel>Content 2</T.Panel>
+      </Hds::Tabs>
+    `);
     assert.dom('#test-tabs').hasClass('hds-tabs');
   });
+
+  test('it should have 2 Tabs and 2 Panels', async function (assert) {
+    await render(hbs`
+      <Hds::Tabs as |T|>
+        <T.Tab>One</T.Tab>
+        <T.Tab>Two</T.Tab>
+        <T.Panel>Content 1</T.Panel>
+        <T.Panel>Content 2</T.Panel>
+      </Hds::Tabs>
+    `);
+    assert.dom('.hds-tabs__tab').exists({ count: 2 });
+    assert.dom('.hds-tabs__panel').exists({ count: 2 });
+  });
+
+  test('it should select the first tab and display the first panel by default', async function (assert) {
+    await render(hbs`
+      <Hds::Tabs as |T|>
+        <T.Tab>One</T.Tab>
+        <T.Tab>Two</T.Tab>
+        <T.Panel>Content 1</T.Panel>
+        <T.Panel>Content 2</T.Panel>
+      </Hds::Tabs>
+    `);
+    assert
+      .dom('.hds-tabs__tab:first-child')
+      .hasClass('hds-tabs__tab--is-selected');
+    assert
+      .dom('.hds-tabs__tab:first-child .hds-tabs__link')
+      .hasAttribute('aria-selected');
+    assert.dom('.hds-tabs__panel:first-of-type').doesNotHaveAttribute('hidden');
+
+    assert
+      .dom('.hds-tabs__tab:not(:first-child)')
+      .doesNotHaveClass('hds-tabs__tab--is-selected');
+    assert
+      .dom('.hds-tabs__tab:not(:first-child) .hds-tabs__link')
+      .doesNotHaveAttribute('aria-selected');
+    assert.dom('.hds-tabs__panel:not(:first-of-type)').hasAttribute('hidden');
+  });
```

Comment:
Suggestion: instead of relying on these CSS selectors that are based on the position of the elements in the DOM, you could alternatively use `data-test` attributes (like it's done here for example: https://github.com/hashicorp/design-system/blob/2f23d6e86587a408cfd62ae05370f279824cf8ed/packages/components/tests/integration/components/hds/form/checkbox/group-test.js#L105-L124) of if you prefer use the DOM `id` attribute (it would override the one generated automatically, but for the purpose of the test would be OK). 

_(If I had to choose, probably I would go with the first one.)_

Reply:
Using data-test attributes is a good idea. Thanks